.prettierrc

```diff
@@ -1,5 +1,10 @@
 {
-  "singleQuote": true,
+  "arrowParens": "avoid",
   "bracketSpacing": false,
-  "quoteProps": "consistent"
+  "printWidth": 120,
```

Comment:
What was the default before? 80?

Reply:
I don’t know, Prettier wrapping rules changed and when I tried with 80 it was introducing a bunch of unwanted line breaks so I chose 120 but I’m open to changing it if people want to keep 80

Reply:
I'm in favor of 120, 80 is a bit too strict I think

Reply:
Yeah, I like 120 too. I was just wondering about the

> keep our defaults

part of the PR description 😉